src/components/organisms/AdminVenueView/components/Container/Container.tsx

```diff
@@ -0,0 +1,317 @@
+import React, {
+  useCallback,
+  useState,
+  useMemo,
+  CSSProperties,
+  Dispatch,
+  SetStateAction,
+  useEffect,
+} from "react";
+import { useDrop } from "react-dnd";
+import update from "immutability-helper";
+import ReactResizeDetector from "react-resize-detector";
+
+import { Dimensions, Position } from "types/utility";
+import { RoomData_v2 } from "types/rooms";
+
+import { DragItem } from "pages/Account/Venue/VenueMapEdition/interfaces";
+import { ItemTypes } from "pages/Account/Venue/VenueMapEdition/ItemTypes";
+import { DraggableSubvenue } from "pages/Account/Venue/VenueMapEdition/DraggableSubvenue";
+import { CustomDragLayer } from "pages/Account/Venue/VenueMapEdition";
+import { snapToGrid as doSnapToGrid } from "pages/Account/Venue/VenueMapEdition/snapToGrid";
+
+const styles: React.CSSProperties = {
+  width: "100%",
+  position: "relative",
+};
+
+export interface RoomIcon {
+  top: number;
+  left: number;
+  url: string;
+  width: number;
+  height: number;
+  roomIndex?: number;
+}
+
+export interface RoomIconsMap {
+  [key: string]: RoomIcon;
+}
+
+interface CoordinatesBoundary {
+  width: number;
+  height: number;
+}
```

Comment:
Since we want to move to a new naming convention, I wonder if we should name these `portals` instead. Wdyt?

Reply:
Yes, would be nice to rename this as well.

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1583/commits/f8fa0c67c34ac626919737b4bbd112b4bd715553